src/main/python/covata/delta/apiclient.py

```diff
@@ -83,7 +81,7 @@ def get_identity(self, requestor_id, identity_id):
         :param str requestor_id: the authenticating identity id
         :param str identity_id: the identity id to retrieve
         :return: the retrieved identity
-        :rtype: dict[str, any]
+        :rtype: dict[str, str]
```

Comment:
are you sure with the return type? if i remember correctly, get identity returns a metadata dict in the json which throws off that `dict[str, str]` type. Another approach is we can return them separately as a tuple of the identity dict and identity metadata dict `(dict[str, str], dict[str, str])`, that will also work but not sure of its correctness..